### DISCUSSION TITLE
Temperature Sensor Input Issues

### DISCUSSION THREAD

**Discussion by subcritical71** [tagyoureit/nodejs-poolController#694] [REM]
I feel like I'm hitting hurdle after hurdle. Thanks for sticking it out with me.

I received my BAS board today and got it connected just fine. 

After trying to add a 2nd temperature probe after successfully installing the first I ran into some issues.

First, the Generic Devices are both set to Raw Value. I tried changing this to Resistance/kohm and it does not stick. I notice I don't have Save and Delete buttons available on the dialog box.

I was going to delete the Generic sensors and start over but ^^ no Delete Device button. I did delete the BAS, rebooted and tried to start over but the Generic Devices are still there. I hate to create more in case it is not possible to delete.

The temperature sensors are working find in the I2C setup and I had the feed working at one point with a single sensor. 

I've tried Mac Safari, Mac Chrome, PC Chrome, and PC Edge, and PC Firefox. None of them give me the Save or Delete Device dialogs.

Is there a way to get this going again? And I forgot to mention, the first probe is no longer working either as it seems that generic device also rested to Raw Value.

**Comment by subcritical71** [comment]
I don't see anything obvious in the config file, unless something should be kohm vs ohm;

```
      {
        "id": 2,
        "values": {
          "adcValue": {
            "id": 2,
            "name": "Water Temperatur",
            "type": "T10k",
            "enabled": true,
            "units": "kohm",
            "ioType": "analog",
            "value": 8.312
          },
          "inputUnits": "kOhms",
          "units": "F",
          "maxVal": 10000,
          "resistance": null,
          "temperature": null
        },
        "options": {
          "name": "Water Temp Probe",
          "units": "F",
          "inputType": "ohms",
          "vccRef": 3.3,
          "inputBitness": 10,
          "inputResistanceUnits": 1000,
          "calibration": 0
        },
        "info": {},
        "triggers": [],
        "feeds": [
          {
            "id": 1,
            "isActive": true,
            "changesOnly": false,
            "sampling": 1,
            "connectionId": 1,
            "sendValue": "temperature",
            "eventName": "temps",
            "property": "waterSensor1"
          }
        ],
        "typeId": 5,
        "isActive": true,
        "name": "Water Temp Probe"
      }
    ]
  },
```

**Reply by subcritical71** [reply]
I ended up getting it working by making two new Generic Sensors and going into he config file and deleting the two sensors that would not go away.

**Reply by tagyoureit** [reply]
Glad you got it working.  Definitely sounds like something went awry.  Next time, look for errors in the Javascript Developer Console (in your browser) and upload the entire `controllerConfig.json` file here and we can investigate.

**Comment by mhmatttu** [comment]
@tagyoureit @rstrouse Happy Memorial Day weekend!  I amhaving this same issue but unable to resolve.  I initially had 64bit raspbian installed and read that may cause issues so started clean with 32bit raspbian.  I updated the firmware on the MEGA-BAS.  Setup of REM, njsPC, and dashPanel per the wiki's all went without issue.  My MEGA-BAS was auto detected by REM.  Added Air Temp as Input.  Created the Generic Device Air Temp Probe for 10k sensor.  No issues with any of these steps.  I go back to the MEGA-BAS and add the Feed.  When I return back to Generic Devices, no temp is reported, tabs at the top for triggers and feeds are gone along with save and delete buttons.  I have replicated the same issue with both versions of rasbpian and by adding multiple generic devices.  Each time I add the MEGA-BAS feed is when the issue arises.

**Reply by mhmatttu** [reply]
Never mind....I did not choose the option with "value" as the wiki clearly says to do!  So I am now getting the feed to Generic Devices, but the resistance is only 0.007 kilo ohms and shows no temp.  Does this mean my sensor is bad or is something else going on?

![image](https://github.com/tagyoureit/nodejs-poolController/assets/47801991/9d1b1f46-82d3-4b3b-b61d-71e5e43f5d8b)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/47801991/1a8233ff-b84f-4f42-bbf1-7833d2a7e470)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/47801991/0a20cf6b-08bb-41cb-9c99-7950d54b4eb4)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/47801991/11fb80be-5854-414b-9a97-4bdd46f408d8)

**Reply by mhmatttu** [reply]
I figured out my problem(s).  The primary issue was that I had the dip switches in the wrong position.  Based on the printed diagram on the board, it appeared to me that switch 1 needed to be on for the 10k thermister, but its in fact switch 3.  The other issue is that i had the generic device units as ohms vs kohms.
